installer/ansible/group_vars/dashboard.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 The OpenSDS Authors.
```

Comment:
This file may not be needed, when dashboard installation is removed

Reply:
done